tests/utils/koa.ts

```
@@ -0,0 +1,63 @@
+      sent: () => void
+    ) => Promise<void> | void,
+    after?: (
+      ctx: ParameterizedContext,
+      next: Next,
+      sent: () => void
+    ) => Promise<void> | void
+  ): Middleware =>
+  async (ctx, next) => {
+    let sent = false
+
+    await before?.(ctx, next, () => (sent = true))
```

Biome (1.9.4)

[error] The assignment should not be in an expression.

The use of assignments in expressions is confusing.
Expressions are often considered as side-effect free.

(lint/suspicious/noAssignInExpressions)